examples/t1.qasm

```diff
@@ -23,8 +23,8 @@ defcal x $q {
    play drive($q), gaussian(100, 30, 5);
 }
 
-for p in [0 : points-1] {
-    for i in [1 : shots] {
+for uint p in [0 : points-1] {
```

Comment:
Wonder why this wasn't caught earlier by https://github.com/openqasm/openqasm/pull/341? Maybe the `defcal` statement earlier matched to the end of the file in the OpenQASM test?

Reply:
Yeah, that'll be exactly the issue.  Now the ANTLR grammar refactor is in place, I'll make a PR to fix those issues.